core/src/test/java/org/elasticsearch/index/query/SpanNearQueryBuilderTest.java

```diff
@@ -63,7 +63,11 @@ public void testValidate() {
         int clauses = randomIntBetween(1, 10);
         for (int i = 0; i < clauses; i++) {
             if (randomBoolean()) {
-                queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                if (randomBoolean()) {
+                    queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                } else {
+                    queryBuilder.clause(null);
```

Comment:
No, this is testing that null-values in list of query builders add validation error. So in order to test this we need to explicitly set it for the test.